docs/elastirini.md

```diff
@@ -76,6 +76,8 @@ sh target/appassembler/bin/IndexCollection -collection JsonCollection -generator
  -es -es.index msmarco-passage -threads 9 -input /path/to/msmarco-passage -storePositions -storeDocvectors -storeRawDocs
 ```
 
+We may need to wait a few minutes after indexing for the index to catch up before performing retrievals, otherwise wrong evaluation metrics are returned.
```

Comment:
retrievals -> retrieval

Reply:
Updated, also lumped in some other replication logs in here too